models/classes/Comment/RdfItemCommentAdapter.php

```
@@ -0,0 +1,102 @@
+    }
+
+    public function countByItemUri(string $itemUri): int
+    {
+        return count($this->findByItemUri($itemUri));
+    }
+
+    private function mapResource(core_kernel_classes_Resource $resource): ItemComment
+    {
+        return new ItemComment(
+            $resource->getUri(),
+            (string) $resource->getOnePropertyValue($this->getProperty(ItemCommentOntology::PROPERTY_ITEM_URI)),
```

PHPStan (2.2.6)

[error] Cannot cast core_kernel_classes_Container to string.


(cast.string)